[Blog] week 12 (#118)
packit-service/packit.dev: content/posts/March-2020.md

[(#472)]: https://github.com/packit-service/packit-service/pull/472
[(#473)]: https://github.com/packit-service/packit-service/pull/473
[(#476)]: https://github.com/packit-service/packit-service/pull/476
[(#453)]: https://github.com/packit-service/packit-service/pull/453

## Week 12 (March 16th - March 20th)

### packit
* When you are executing `copr-build` command, you do not 
  need to set the project name if this value is defined in
  the copr-build job in the configuration file [(#763)]

> * Packit orders all patches that are generated when adding them into a specfile [(#765)]

Reviewer: Yeah, the fix I did doesn't work and is not changing anything in the end.

Let's either write
```
All patches generated from a source-git repo are now prefixed with a number so they are easy to sort.
```

or remove the line. Thanks.

* We improved the behaviour when loading the authentication in the config file - 
  users are warned only if deprecated keys are used, no more confusing messages when 
  you do not have `authentication` key in the configuration [(#754)]

### packit-service
* We periodically check the status of the copr-build so that we do not need to rely
  on the result message, this should avoid failures in reporting the status of build [(#490)]
* `Propose-update` triggered by comment in Github now recognizes also pre-releases [(#498)]
* We have updated the targets in Makefile so you can now run requre tests without

>   Openshift [(#497)]

Reviewer 2: end users probably don't care about our `Makefile`

> * Testing farm in Packit-service works again, we fixed several bugs 

Reviewer 2: ```suggestion
Several bugs in how we report results from Testing farm have been fixed
```

Reviewer: +"...and testing farm results should now appear correctly in commit status checks"
